### PR TITLE
adding table to the tutorial-list component

### PR DESCRIPTION
**Pull request by amirdedeic**

### CODE REVIEW

File: project_frontend/Anguler_Frontend/src/app/components/tutorial-list/tutorial-list.component.html [hunk 1]
```diff
@@ -1,6 +1,8 @@
...
-
-    
-
-</p>
+<h1> HELLOOOOO</h1>
 
+<table>
+  <tr *ngFor="let tutorial of tutorial_list; let i = index">
+    <td>{{ tutorial.title }}</td>
+    <td>{{ tutorial.author }}</td>
```
Comment (nacongo) [inline project_frontend/Anguler_Frontend/src/app/components/tutorial-list/tutorial-list.component.html:6]: You need to add date published here as well.

File: project_frontend/Anguler_Frontend/src/app/components/tutorial-list/tutorial-list.component.html [hunk 2]
```diff
@@ -1,6 +1,8 @@
-<p>tutorial-list works!
-
-    
-
-</p>
+<h1> HELLOOOOO</h1>
```
Comment (nacongo) [inline project_frontend/Anguler_Frontend/src/app/components/tutorial-list/tutorial-list.component.html:1]: You can remove unnecessary code.

File: project_frontend/Anguler_Frontend/src/app/components/tutorial-list/tutorial-list.component.ts [hunk 3]
```diff
@@ -9,4 +11,20 @@ import { Component } from '@angular/core';
...
 export class TutorialListComponent {
 
+  tutorial_list: Tutorial[] = [];
+
+  
+  constructor(private tutorialService: TutorialService) {
+    this.tutorialService.getAll()
+      .subscribe(
+        data => {
+          console.log(data)
```
Comment (nacongo) [inline project_frontend/Anguler_Frontend/src/app/components/tutorial-list/tutorial-list.component.ts:21]: You can remove this `console.log()`. It was added just for debugging purpose.

**Review by nacongo — CHANGES_REQUESTED** [review]
Nice work! 👏
I've added few comments. You can fix them and then you're good to go. 🚢

File: project_frontend/Anguler_Frontend/src/app/Models/tutorial.ts [hunk 4]
```diff
@@ -0,0 +1,12 @@
+export class Tutorial { 
+    title: string = '';
```
[the remark this answers is not in the document]
Comment (amirdedeic) [reply project_frontend/Anguler_Frontend/src/app/Models/tutorial.ts:2]: should the initial value be set to 0?

**Pull request merged**